docs/verify/__use_estimator.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: Estimator primitive
+description: How to compute an expectation value with the `Estimator` primitive
+---
+
+# Compute an expectation value with `Estimator` primitive
+
+This guide shows how to get the expected value of an observable for a given quantum circuit with the [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) primitive.
+
+<Admonition type="note" title="Note">
+  While this guide uses Qiskit’s reference implementation, the `Estimator` primitive can be run with any provider using [`qiskit.primitives.BackendEstimator`](../api/qiskit/qiskit.primitives.BackendEstimator).
+
+  ```python
+  from qiskit.primitives import BackendEstimator
+  from <some_qiskit_provider> import QiskitProvider
+
+  provider = QiskitProvider()
+  backend = provider.get_backend('backend_name')
+  estimator = BackendEstimator(backend)
+  ```
+
+  There are some providers that implement primitives natively (see [this page](https://qiskit.org/providers/#primitives) for more details).
+</Admonition>
+
+## Initialize observables
+
+The first step is to define the observables whose expected value you want to compute. Each observable can be any `BaseOperator`, like the operators from [`qiskit.quantum_info`](../api/qiskit/quantum_info).
+Among them it is preferable to use [`qiskit.quantum_info.SparsePauliOp`](../api/qiskit/qiskit.quantum_info.SparsePauliOp).
+
+```python
+from qiskit.quantum_info import SparsePauliOp
+
+observable = SparsePauliOp(["II", "XX", "YY", "ZZ"], coeffs=[1, 1, -1, 1])
+```
+
+## Initialize quantum circuit
+
+Then you need to create the [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit)s for which you want to obtain the expected value.
+
+```python
+from qiskit import QuantumCircuit
+
+qc = QuantumCircuit(2)
+qc.h(0)
+qc.cx(0,1)
+qc.draw("mpl")
+```
+
+![Initial QuantumCircuit](/images/verify/use_estimator/initialize.png)
+
+<Admonition type="note" title="Note">
+  The [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit) you pass to [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) must not include any measurements.
+</Admonition>
+
+## Initialize the `Estimator`
+
+Then, you need to instantiate an [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator).
+
+```python
+from qiskit.primitives import Estimator
+
+estimator = Estimator()
+```
+
+## Run and get results
+
+Now that you have defined your `estimator`, you can run your estimation by calling the [`qiskit.primitives.Estimator.run`](../api/qiskit/qiskit.primitives.Estimator#run) method,
+which returns an instance of [`qiskit.providers.JobV1`](../api/qiskit/qiskit.providers.JobV1). You can get the results from the job (as a [`qiskit.primitives.EstimatorResult`](../api/qiskit/qiskit.primitives.EstimatorResult) object)
```

Comment:
I changed this sentence! It used to say it returned a `PrimitiveJob` which is a subclass of `JobV1`. But I couldn't find that class anymore. So I instead simply now say it returns a `JobV1`.